extension/src/ToolTipsFunctions.ts

```diff
@@ -310,11 +311,16 @@ export function addSuggestedTooltips(alObject: ALObject) {
 
     function getToolTipFromOtherPages(control: ALControl) {
         let toolTip;
-        let pageObjects = alObject.alObjects?.filter(obj => obj.sourceTable === alObject.sourceTable && (obj.objectType === ALObjectType.Page || obj.objectType === ALObjectType.PageExtension) && !(obj.objectType === alObject.objectType && obj.objectId === alObject.objectId));
+        let pageObjects = alObject.alObjects?.filter(obj => obj.sourceTable === alObject.sourceTable &&
+            (obj.objectType === ALObjectType.Page ||
+                obj.objectType === ALObjectType.PageExtension) &&
```

Comment:
Maybe not really in the scope of this PR but I would be quick to forget it if let it pass :)
```suggestion
            [ALObjectType.Page, ALObjectType.PageExtension].includes(obj.objectType) &&
```